code/game/objects/items/weapons/material/material_armor_VR.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/weapon/material/armor_plating/insert
+	unbreakable = FALSE
+	name = "plate insert"
+	desc = "used to craft armor plates for a plate carrier. Trim with a welder for light armor or add a second for heavy armor"
```

Comment:
This can't be here, it was already added in another PR from upstream presumably by you, so no need for this